package.json

```diff
@@ -48,7 +48,7 @@
     "react-component"
   ],
   "scripts": {
-    "test": "mocha --compilers @babel/register --require testHelper.js \"./{,!(node_modules)/**/}/__test__/*.js\"",
+    "test": "mocha --compilers @babel/register --require testHelper.js \"./*/src/**/__test__/*.js\"",
```

Comment:
Isn't this change unnecessary?

Reply:
Yes, because without this we run tests from lib folder.

Reply:
kk